app/packages/components/package.json

```diff
@@ -46,6 +46,7 @@
         "@textea/json-viewer": "^3.4.1",
         "classnames": "^2.3.1",
         "framer-motion": "^6.2.7",
+        "material-icons": "^1.13.12",
```

Comment:
Why are we bringing it in when we already have `@mui/icons-material`? This will cause significant bloating otherwise.

Reply:
This is font-based and it's to support dynamic icon by name. In a separate PR, we should probably replace all the icons from `@mui/icons-material` to this instead. It will reduce build size/time